lib/lighthouse_bgs/base.rb

```diff
@@ -141,7 +145,7 @@ def client
         convert_request_keys_to: :none,
         ssl_verify_mode: @ssl_verify_mode.to_sym
       }
-      options[:host] = @forward_proxy_url.gsub(%r{^https?://}, '') unless @forward_proxy_url.nil?
+      options[:endpoint] = endpoint unless @forward_proxy_url.nil?
```

Comment:
Are you basically feeding the end point back to gem after it's pulled down from the wsdl essentially or is it the end point pre wsdl?

Reply:
They are 2 different options passed into the Savin client constructor however they are very similar urls

Reply:
Savon allows the endpoint setting be the first option for all non-wsdl requests. So in the same way the wsdl url was being built with the service_name and bean name, this does the same without the wsdl params at the end.